README.md

```diff
@@ -13,13 +13,15 @@ After finishing this masterclass you will have basic understanding of C# and .NE
 Please follow [flipped classroom](FLIPPED_CLASSROOM.md).
 
 ## Planning
+
 | Week | Topic                                                        | Preparation                         | Lesson plan                         | Homework                      |
 | ---- | ------------------------------------------------------------ | ----------------------------------- | ----------------------------------- | ----------------------------- |
-| 1.   | **Fundamentals** <br />* Intro to .NET and C#<br />* First program (console)<br />* Data types and variables<br />* Methods<br />* Controlling program flow | [Preparation](week1/preparation.md) | [Lesson plan](week1/lesson-plan.md) | [Homework](week1/homework.md) |
+| 1.   | **Fundamentals** <br />* Intro to .NET and C#<br />* First program (console)<br />* Data types and variables<br />* Methods<br />* Controlling program flow<br />* Debugging program | [Preparation](week1/preparation.md) | [Lesson plan](week1/lesson-plan.md) | [Homework](week1/homework.md) |
 | 2.   | **C# Fundamentals**<br />* Intro to OOP concepts<br />* Classes<br />* Abstract classes | [Preparation](week2/preparation.md) | [Lesson plan](week2/lesson-plan.md) | [Homework](week2/homework.md) |
-| 3.   | **C# Fundamentals**<br />* Interfaces<br />* Async programming | [Preparation](week3/preparation.md) | [Lesson plan](week3/lesson-plan.md) | [Homework](week3/homework.md) |
-| 4.   | **Working with database**<br />* [ADO.NET](http://ado.net/) introduction<br />* Connecting and querying data<br />* Reading and persisting data | [Preparation](week4/preparation.md) | [Lesson plan](week4/lesson-plan.md) | [Homework](week4/homework.md) |
-| 5.   | **[ASP.NET](http://asp.net/)** Core<br />* Anatomy of [asp.net](http://asp.net/) core application<br />* Dependency Injection<br />* Minimal API<br />* Building first API (creating first controllers and models) | [Preparation](week5/preparation.md) | [Lesson plan](week5/lesson-plan.md) | [Homework](week5/homework.md) |
+| 3.   | **[ASP.NET](http://asp.net/) Core fundamentals**<br />* Anatomy of the [ASP.NET](http://asp.net/) core application<br />* Controllers<br />* Models | [Preparation](week3/preparation.md) | [Lesson plan](week3/lesson-plan.md) | [Homework](week3/homework.md) |
```

Comment:
Anatomy of the ASP.NET core application (Program, Startup, other
Controllers  - Routes, Verbs etc
Models - Request, Response, Model Binding
... some other stuff that I probably missed